### DISCUSSION TITLE
lack of example of Client Proxy to Separate Server Proxies Pools

### DISCUSSION THREAD

**Discussion by liuqianhong6007** [googleforgames/quilkin#422] [Q&A]
According to the guide, I know there is three method to integrate the quilkin, but I can't find an example about how to use quilkin as client proxy and server proxy at the same time.  I wonder how the client proxy communicate with server proxy and how to configure it for the target. Appreciate it

**Comment by markmandel** [comment]
While not a proxy pool example, this example does show a client and server proxy working in conjunction:

https://github.com/googleforgames/quilkin/tree/main/examples/agones-xonotic#sidecar-with-compression-filter

**Comment by liuqianhong6007** [comment]
> While not a proxy pool example, this example does show a client and server proxy working in conjunction:
> 
> https://github.com/googleforgames/quilkin/tree/main/examples/agones-xonotic#sidecar-with-compression-filter
Thanks for reply. I am confused by the client proxy configuration if I want to use seperate server proxies.  Should client proxy's endpoint address relate to one server proxy address ?  And in this mode, quilkin shoud run as a independant service, right ?

**Comment by iffyio** [comment]
>Thanks for reply. I am confused by the client proxy configuration if I want to use seperate server proxies. Should client proxy's endpoint address relate to one server proxy address ? And in this mode, quilkin shoud run as a independant service, right ?

Hi @liuqianhong6007 I think this would depend mostly on how your server proxies are setup. the client proxy likely wants to send packets to any one of those servers then it needs to know all of their addresses - that information can come either statically from the yaml configuration file or from a [management server](https://googleforgames.github.io/quilkin/main/book/xds.html).

**Comment by XAMPPRocky** [comment]
Going to move this over to a discussion, since it's more of a question on how to use Quilkin.
